test/gcloud/helpers/cds.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// Provides a global function for clearing a Cloud Datastore database. This is
+// necessary for giving a clean slate to each Cloud Datastore test.
+//
+
+var env = require('process').env;
+var http = require('http');
+
+require('./foam.js');
+
+global.clearCDS = function() {
```

Comment:
This works out fine. `../../../src/foam.js` will be included exactly once, after `global.FOAM_FLAGS = { debug: true, gcloud: true };`. Either it gets required here, or it gets required when `./foam.js` is loaded before this helper.

If I want to be more paranoid about how the `require()`s behave, I could use `__dirname/` instead of `./` though.